recipe/meta.yaml

```
@@ -22,10 +22,10 @@ requirements:
     - make
     - ninja
   host:
-    - mbedtls  # [not win]
+    - mbedtls =3.5
   run:
     - ucrt  # [win]
-    - mbedtls  # [not win]
+    - mbedtls  =3.5
```

We could pin to the version which was used during building, using: 
```suggestion
    - {{ pin_compatible('mbedtls', max_pin='x.x.x') }}
```
https://conda-forge.org/docs/maintainer/pinning_deps.html

---

Ah, I missed this in the pinning documentation.  That seems ideal.